starknet_py/cairo/serialization/data_serializers/uint256_serializer_test.py

```diff
@@ -33,28 +36,61 @@ def test_valid_values(value, serialized_value):
     serialized = serializer.serialize(value)
     assert serialized == serialized_value
 
+    assert serialized_value == serializer.serialize(
+        {"low": serialized_value[0], "high": serialized_value[1]}
+    )
+
 
-def test_invalid_ranges():
+def test_deserialize_invalid_values():
     # We need to escape braces
     low_error_message = re.escape(
         "Error at path 'low': expected value in range [0;2^128)"
     )
-    high_error_message = re.escape(
-        "Error at path 'high': expected value in range [0;2^128)"
-    )
     with pytest.raises(InvalidValueException, match=low_error_message):
         serializer.deserialize([MAX_U128 + 1, 0])
-    with pytest.raises(InvalidValueException, match=high_error_message):
-        serializer.deserialize([0, MAX_U128 + 1])
     with pytest.raises(InvalidValueException, match=low_error_message):
         serializer.deserialize([MAX_U128 + 1, MAX_U128 + 1])
     with pytest.raises(InvalidValueException, match=low_error_message):
         serializer.deserialize([-1, 0])
+
+    high_error_message = re.escape(
+        "Error at path 'high': expected value in range [0;2^128)"
+    )
+    with pytest.raises(InvalidValueException, match=high_error_message):
+        serializer.deserialize([0, MAX_U128 + 1])
     with pytest.raises(InvalidValueException, match=high_error_message):
         serializer.deserialize([0, -1])
 
+
+def test_serialize_invalid_int_value():
     error_message = re.escape("Error: Uint256 is expected to be in range [0;2^256)")
```

Comment:
Same here

Reply:
There are a few more places where `^` is used. Let's change that too